sdk/storage/azure-storage-file-datalake/README.md

```diff
@@ -25,17 +25,20 @@ pip install azure-storage-file-datalake --pre
 
 ### Create a storage account
 If you wish to create a new storage account, you can use the
-[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-the-azure-portal),
-[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-powershell),
-or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-azure-cli):
+[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal),
```

Comment:
why do we need to change the link? The existing link is under "How to" under ADLS Gen2, after it's changed it's under blob storage.

Reply:
These two links point to the same page. You can open it and have a look.

Reply:
but the tabs are different right?

Reply:
I have updated "Azure Portal" link.

Reply:
Could you please revert the changes for powershell and azure CLI as well? Or is there's a reason you want to change those two?

Reply:
About Powershell and azure CLI , after the two links are opened, the corresponding pages are all about the steps of azure portal. You can have a try. The reason why I changed it is to refer to the SDK in other languages, like [net sdk storage file datalake readme](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Files.DataLake#prerequisites).